config/db

```diff
@@ -21,10 +20,10 @@ CentOS_5_ruby_1.9.2_patch_level=p320
 CentOS_5_ruby_1.9.2_patch_level_warn=Warning, Centos 5 can not build head version of ruby, falling back to latest patchlevel, your ruby will be less secure bacuase of this.
 CentOS_5_ruby_1.8.7_patch_level=p374
 CentOS_5_ruby_1.8.7_patch_level_warn=Warning, Centos 5 can not build head version of ruby, falling back to latest patchlevel, your ruby will be less secure bacuase of this.
-rubygems_url=http://production.cf.rubygems.org/rubygems
+rubygems_url=https://rubygems.global.ssl.fastly.net/rubygems
```

Comment:
This is what the rubygems.org folk recommended when I asked about SSL for the CloudFront instance (which is what production.cf.rubygems.org uses). Also, they are deprecating CloudFront soon anyway (replacing it with Fastly completely, though maybe something else in the future).